packages/example-app-base/src/components/ExampleCard/ExampleCard.tsx

```diff
@@ -168,7 +168,11 @@ export class ExampleCardBase extends React.Component<IExampleCardProps, IExample
               </div>
               {isCodeVisible ? (
                 <EditorWrapper
-                  useEditor={false}
+                  // Disable editor for list examples until crash can be figured out
+                  useEditor={
```

Comment:
Could we just default to disabled unless useEditor=1 is set in the QS param? At least until we know why it's failing.

Reply:
I still disagree about this but have reverted it

Reply:
Thanks. We'll get it back online fully once we know why it is crashing the browser.